### PR TITLE
Opal 242 events into merged kalendar

### PR DESCRIPTION
**Pull request by markaron**

### CODE REVIEW

File: src/main/java/com/greenfoxacademy/kalendaryo/service/KalendarService.java [hunk 1]
```diff
@@ -127,6 +128,7 @@ public Kalendar setKalendarAttribute(Kalendar kalendar, KalendarFromAndroid kale
         } else {
             kalendar.setName(kalendarFromAndroid.getCustomName());
         }
+
```
Comment (bekobarna) [inline src/main/java/com/greenfoxacademy/kalendaryo/service/KalendarService.java]: why the empty line?

File: src/main/java/com/greenfoxacademy/kalendaryo/service/authorization/AuthorizeKal.java [hunk 2]
```diff
@@ -103,56 +103,93 @@ public TokenResponse authorizeWithRefreshToken(String refreshToken) throws IOExc
...
     }
 
+    private void migrateEvents(GoogleCalendarFromAndroid[] sourceCalendarIds , String googleCalendarId) throws IOException {
+        for (int i = 0; i < sourceCalendarIds.length; i++) {
+            buildCalendarFromAndroidClient(sourceCalendarIds[i].getEmail());
+            String visibility = setVisibility(sourceCalendarIds[i]);
+            String calendarId = sourceCalendarIds[i].getId();
+            String pageToken = null;
+            do {
+                Events events = calendarFromAndroidClient.events().list(calendarId).setPageToken(pageToken).execute();
```
Comment (bekobarna) [inline src/main/java/com/greenfoxacademy/kalendaryo/service/authorization/AuthorizeKal.java]: we could use new lines for easier readability, like at line 79

**Review by bekobarna — DISMISSED** [review]

File: src/test/java/com/greenfoxacademy/kalendaryo/service/authorization/AuthorizeKalTest.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.io.IOException;
+
+import static com.greenfoxacademy.kalendaryo.service.authorization.AuthorizeKal.NO_MERGED_CALENDAR;
+import static org.junit.Assert.*;
+
+public class AuthorizeKalTest {
+
+  private MockMvc mock;
```
Comment (bekobarna) [inline src/test/java/com/greenfoxacademy/kalendaryo/service/authorization/AuthorizeKalTest.java]: we dont use this

Comment (markaron) [reply src/test/java/com/greenfoxacademy/kalendaryo/service/authorization/AuthorizeKalTest.java]: done

**Review by sznemeth3 — APPROVED** [review]

**Review by bekobarna — APPROVED** [review]

**Pull request merged**